tutorials/getting-started-netcore.rst

```diff
@@ -0,0 +1,186 @@
+Getting Started on .NET Core
+============================
+
+.. note::
+
+    Still on the .NET Framework? Check out our alternate guide to :doc:`/tutorials/getting-started` on the .NET Framework.
+
+You can be up and running with Freya in minutes! This quick guide will take you
+from zero to Hello World, using some of the high level building blocks that
+Freya provides out of the box -- and tell you where to go to learn more about
+each of them.
+
+Your Hello World will be completely self contained, and able to greet
+individuals by name if they provide one -- reverting to a classic Hello World if
+not. It's certainly possible to write this in a simpler way using Freya, but
+this implementation introduces some of the key Freya building blocks which will
+be valuable as you build more complex applications.
+
+Before we get started, you will want to install the Freya project templates for
+.NET Core. You can do this using the :program:`dotnet` CLI tool.
+
+.. code-block:: shell
+
+    dotnet new --install Freya.Template::*
+
+This process will take a few minutes to install the templates on your host,
+after which, you can create a Freya project by simply typing
+
+.. code-block:: shell
+
+    dotnet new freya
+
+.. hint::
+
+    By default, the Freya template will use the ``Async`` construct from ``FSharp.Core``. If you want to use the ``Job`` construct from `Hopac <https://hopac.github.io/Hopac/Hopac.html>`_, then create the project using ``dotnet new freya -f hopac``.
+
+This will create a basic Freya project for you, including four files. :file:`Api.fs`,
+:file:`KestrelInterop.fs`, :file:`Program.fs`, and a :file:`{Freya}.fsproj` file.
+
+First, let's take a look at :file:`Api.fs`. Here's the complete code,
+before the implementation is broken down:
+
+.. code-block:: fsharp
+
+    module Api
+
+    open Freya.Core
+    open Freya.Machines.Http
+    open Freya.Types.Http
+    open Freya.Routers.Uri.Template
+
+    let routeName_ = Route.atom_ "name"
+
+    let name = freya {
+      let! nameO = Freya.Optic.get routeName_
+
+      match nameO with
+      | Some name -> return name
+      | None -> return "World"
+    }
```

Comment:
Pretty much everywhere in Freya source and docs i've gone with the closing brace on the last line style. I'd prefer to keep that consistent if possible? It aids concision a bit I feel.